sample-input/benchmarks/LRA/LRA-cmfd.py

```diff
@@ -82,98 +83,189 @@
-                           [4, 4, 4, 4, 4, 4, 4, 4, 4, 4],
-                           [4, 4, 4, 4, 4, 4, 4, 4, 4, 4]])
-
-assembly5 = Lattice(id=35, width_x=1.5, width_y=1.5)
-assembly5.setLatticeCells([[5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-                           [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-                           [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-                           [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-                           [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-                           [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-                           [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-                           [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-                           [5, 5, 5, 5, 5, 5, 5, 5, 5, 5],
-                           [5, 5, 5, 5, 5, 5, 5, 5, 5, 5]])
-
-
-assembly6 = Lattice(id=36, width_x=1.5, width_y=1.5)
-assembly6.setLatticeCells([[6, 6, 6, 6, 6, 6, 6, 6, 6, 6],
-                           [6, 6, 6, 6, 6, 6, 6, 6, 6, 6],
-                           [6, 6, 6, 6, 6, 6, 6, 6, 6, 6],
-                           [6, 6, 6, 6, 6, 6, 6, 6, 6, 6],
-                           [6, 6, 6, 6, 6, 6, 6, 6, 6, 6],
-                           [6, 6, 6, 6, 6, 6, 6, 6, 6, 6],
-                           [6, 6, 6, 6, 6, 6, 6, 6, 6, 6],
-                           [6, 6, 6, 6, 6, 6, 6, 6, 6, 6],
-                           [6, 6, 6, 6, 6, 6, 6, 6, 6, 6],
-                           [6, 6, 6, 6, 6, 6, 6, 6, 6, 6]])
-
-
-core = Lattice(id=7, width_x=15.0, width_y=15.0)
-core.setLatticeCells([[26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26],
-                         [26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26],
-                         [23, 23, 23, 23, 23, 23, 23, 26, 26, 26, 26],
-                         [23, 23, 23, 23, 23, 23, 23, 24, 26, 26, 26],
-                         [22, 21, 21, 21, 21, 22, 22, 25, 25, 26, 26],
-                         [22, 21, 21, 21, 21, 22, 22, 25, 25, 26, 26],
-                         [21, 21, 21, 21, 21, 21, 21, 23, 23, 26, 26],
-                         [21, 21, 21, 21, 21, 21, 21, 23, 23, 26, 26],
-                         [21, 21, 21, 21, 21, 21, 21, 23, 23, 26, 26],
-                         [21, 21, 21, 21, 21, 21, 21, 23, 23, 26, 26],
-                         [22, 21, 21, 21, 21, 22, 22, 23, 23, 26, 26]])
+log.py_printf('NORMAL', 'Creating LRA lattices...')
+
+assembly1 = Lattice(name='assembly 1')
+assembly1.setWidth(width_x=1.5, width_y=1.5)
+template = [[1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1]]
+
+for i in range(10):
+  for j in range(10):
+    template[i][j] = universes[template[i][j]-1]
```

Comment:
I agree with Sam that the 0-based indexing in the lists compared to the 1-based indexing in the template is a little weird.  I don't see a wildly better option regardless.